distributed/nanny.py

```diff
@@ -263,9 +261,8 @@ async def start(self):
         )
         self.ip = get_address_host(self.address)
 
-        await preloading.on_start(
-            self._preload_modules, self, argv=self.preload_nanny_argv,
-        )
+        for preload in self.preloads:
+            await preload.start()
```

Comment:
Do you want these to run serially, or could the be done concurrently (e.g. using `asyncio.gather`)?

Reply:
If a call to `preload.start` fails, this will cause the scheduler startup to also fail, correct?

Reply:
Making them sequential was intentional.  I figured it was more likely that someone would complain about preloads interacting poorly with each other than they would care about preloads being slow because they're not being run concurrently.

And yes, currently things will fail if a preload fails.  I think that I'm ok with that.

Reply:
> And yes, currently things will fail if a preload fails. I think that I'm ok with that.

I think that's the right behavior.